src/assets/scripts/client/App.js

```diff
@@ -145,8 +141,6 @@ export default class App {
      * @method enable
      */
     enable() {
-        console.info(`openScope Air Traffic Control Simulator, Version v${this.prop.version}`);
```

Comment:
You don't think this is handy to have? Once they pass the loading screen, the only way to be sure a user doesn't have an old version cached accidentally is to run the `version` command, which won't work if the app crashed for some reason.